gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/step/map/PropertyMapStep.java

```diff
@@ -248,4 +251,16 @@ protected void applyTraversalRingToMap(Map<Object, Object> map){
             this.traversalRing.reset();
         }
     }
+
+    public int getTokens() {
```

Comment:
I think this duplicates getIncludedTokens().

Reply:
Thanks for pointing on that! I auto-generated those getter methods and missed that `getIncludedTokens()` already exists. I will remove this auto-generated method

Reply:
Done